coffee_core/src/coffee.rs

```diff
@@ -297,6 +297,21 @@ impl PluginManager for CoffeeManager {
         self.storage.store(&self.storage_info()).await?;
         Ok(())
     }
+
+    async fn show(&mut self, plugin: &str) -> Result<String, CoffeeError> {
+        for repo in &self.repos {
+            if let Some(plugin) = repo.get_plugin_by_name(plugin) {
+                let readme_path = format!("{}/README.md", plugin.path);
+                let contents = fs::read_to_string(readme_path)?;
+                return Ok(contents);
```

Comment:
```suggestion
                return Ok(json!{ "show": contents });
```

The core need to return always a json otherwise you return different kind of data for different command and this make the code a mess!

Reply:
Thanks for pointing that out